minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -356,14 +357,13 @@ private ProcessInfo _exec(Class<?> clazz, List<String> extraJvmOpts, String... a
     var hardcodedArgs = Stream.of(
         "-Dapple.awt.UIElement=true",
```

Comment:
Why not add these two options to the set of defaults?  Then could have no hard coded jvm options and all jvm options can be mutated.

Reply:
I think technically those are system properties.

Reply:
I got rid of the rest of the hardcoded args in 348d747. I added them to the system properties map in the Config class.